sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -15,9 +16,15 @@ namespace Azure.Core.TestFramework
 {
     public abstract class RecordedTestBase : ClientTestBase
     {
+        static RecordedTestBase()
+        {
+            // remove once https://github.com/Azure/azure-sdk-for-net/pull/25328 is shipped
+            ServicePointManager.Expect100Continue = false;
```

Comment:
You started adding azure.core references to individual projects. Can we take this out now?

Reply:
I'd rather wait as removing this will make libraries that I didn't need to update start failing.

Reply:
Good with me